tests/integration/helpers.py

```diff
@@ -423,6 +424,19 @@ def applications(self) -> Mapping[str, dict]:
         """
         return self.content["applications"]
 
+    @property
+    def needs_trust(self) -> bool:
+        """Check if the bundle needs to be trusted.
+
+        Returns:
+            bool: True if the bundle needs to be trusted
+        """
+        return any(
+            app.get("trust", False)
+            for app in self.content["applications"].values()
```

Comment:
nit: Maybe an overly defensive programmer here:
```suggestion
            for app in self.content.get("applications", {}).values()
```

Reply:
Alright sure.  Check out the new impl